### PR TITLE
Affiliate page

### LINKED ISSUE #20: Add affiliate army program
https://kloud51.com/index.php?slug=affiliate-army-program

### LINKED ISSUE #24: Add Affiliate Army Program section A.S.A.P.

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Resolves #20

### CODE REVIEW

File: app/affiliate.html [hunk 1]
```diff
@@ -137,14 +128,17 @@
     <hr>
     <!-- Links -->
     <div class="row">
-      <div class="col-md-8 col-md-offset-2">
-        <p class="text-center">
+      <div class="col-md-4 text-center">
+        <i class="fa fa-link fa-6x o-fade text-primary"></i>
+      </div>
+      <div class="col-md-8">
+        <p>
```
Comment (AmirSavand) [inline app/affiliate.html]: This can be one line.

File: app/affiliate.html [hunk 2]
```diff
@@ -137,14 +128,17 @@
...
           In order to send potential customers to our web-site and have them listed as one of your referrals,              
         </p>
-        <p class="text-center">
+        <p>
```
Comment (AmirSavand) [inline app/affiliate.html]: One line.

**Review by AmirSavand — CHANGES_REQUESTED** [review]

File: app/affiliate.html (hunk 1 above)
Comment (ArsalanSavand) [reply app/affiliate.html]: Done

File: app/affiliate.html (hunk 2 above)
Comment (ArsalanSavand) [reply app/affiliate.html]: Done.

**Review by AmirSavand — APPROVED** [review]

**Pull request merged**